CRM/Contribute/BAO/Contribution.php

```diff
@@ -1190,13 +1190,13 @@ private static function createFinancialItemsForLine($params, $context, $fields,
         $receiveDate = CRM_Utils_Date::isoToMysql($params['contribution']->receive_date);
       }
 
-      $financialAccount = self::getFinancialAccountForStatusChangeTrxn($params, CRM_Utils_Array::value('financial_account_id', $prevFinancialItem));
+      $financialAccount = self::getFinancialAccountForStatusChangeTrxn($params, $prevFinancialItem['financial_account_id'] ?? NULL);
 
       $currency = $params['prevContribution']->currency;
       if ($params['contribution']->currency) {
         $currency = $params['contribution']->currency;
       }
-      $previousLineItemTotal = CRM_Utils_Array::value('line_total', CRM_Utils_Array::value($fieldValueId, $previousLineItems), 0);
+      $previousLineItemTotal = $previousLineItems[$fieldValueId]['line_total'] ?? 0;
```

Comment:
Technically when there's a default value passed in it's not a 1-1 swap with `??`. Consider `cv ev "$a['b'] = NULL; var_dump(CRM_Utils_Array::value('b', $a, 4)); var_dump($a['b'] ?? 4);"`. The first outputs null and the second outputs 4. So to review you need to think about whether null would be possible/make sense in the original. For these `0` ones I think they're ok.